sparkle/src/components/DataTable.tsx

```diff
@@ -416,6 +432,25 @@ DataTable.MoreButton = function MoreButton({
               }}
             />
           ))}
+          {submenus?.map((submenu) => (
```

Comment:
So submenus are always after moreMenuItems ? Can't we extends moreMenuItems to support both and have one single list we can order as we want ?

Reply:
@tdraier revised the approach, how about now?